Bluepill-runner/Bluepill-runner/BPRunner.m

```diff
@@ -313,11 +313,24 @@ - (int)runWithBPXCTestFiles:(NSArray<BPXCTestFile *> *)xcTestFiles {
         if ([fm fileExistsAtPath:outputPath]) {
             [fm removeItemAtPath:outputPath error:nil];
         }
-        [BPReportCollector collectReportsFromPath:self.config.outputDirectory onReportCollected:^(NSURL *fileUrl) {
+        [BPReportCollector collectReportsFromPath:self.config.outputDirectory applyXQuery:nil onReportCollected:^(NSURL *fileUrl) {
 //            NSError *error;
 //            NSFileManager *fm = [NSFileManager new];
 //            [fm removeItemAtURL:fileUrl error:&error];
         } outputAtPath:outputPath];
+        if (self.config.failureXmlOutput) {
+            outputPath = [self.config.outputDirectory stringByAppendingPathComponent:@"FailureReport.xml"];
+            [BPReportCollector collectReportsFromPath:self.config.outputDirectory applyXQuery:@"testcase/failure[@message!=\"Timed out waiting for the test to produce output. Test was aborted.\"]" onReportCollected:^(NSURL *fileUrl) {} outputAtPath:outputPath];
```

Comment:
imeouts? shouldn't this be crashes? How is it different from the `timeoutXmlOutput` below?

Reply:
This is looking for all failures that don't have the timeout message (Its !=, not =.) I was hesitant to check the duration for greater than the config timeout because of some bugs with the timeout detection I've seen when looking at voyager logs, so I did this to at least grab all tests that were marked at timeouts by Bluepill.

Reply:
Ah, I missed the `!`. Thanks.